packages/blaze/template.js

```diff
@@ -318,6 +322,23 @@ Template.prototype.helpers = function (dict) {
     this.__helpers.set(k, dict[k]);
 };
 
+// Kind of like Blaze.currentView but for the template instance.
+// This is a function, not a value -- so that not all helpers
+// are implicitly dependent on the current template instance.
```

Comment:
The reason we made this change was that Template.instance() actually does have a Tracker dependency on the data context, so this function was being re-run every time the data context changed.